### PR TITLE
Updated release version to 4.53.0 for azure-cosmos, 4.24.0 for spark connector and 2.7.0 for azure-cosmos-encryption

### PR DESCRIPTION
**Pull request by kushagraThapar**
Updated release version to 4.53.0 for azure-cosmos, 4.24.0 for spark connector and 2.7.0 for azure-cosmos-encryption

### CODE REVIEW

File: sdk/cosmos/azure-cosmos-spark_3-1_2-12/README.md [hunk 1]
```diff
@@ -131,6 +134,7 @@ https://github.com/Azure/azure-sdk-for-java/issues/new
 #### azure-cosmos-spark_3-4_2-12
 | Connector | Supported Spark Versions | Minimum Java Version | Supported Scala Versions | Supported Databricks Runtimes |
 |-----------|--------------------------|----------------------|--------------------------|-------------------------------|
+| 4.24.0    | 3.4.0                    | 8                    | 2.12                     | 11.\*, 12.\*                  |
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos-spark_3-1_2-12/README.md:137]: This is wrong - 13.* is Spark 3.4 - probably worth fixing in this PR? Same for other SPark versions

Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos-spark_3-1_2-12/README.md]: Done, thanks!

File: sdk/cosmos/azure-cosmos-spark_3_2-12/pom.xml [hunk 2]
```diff
@@ -63,7 +63,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-cosmos</artifactId>
```
[the remark this answers is not in the document]
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos-spark_3_2-12/pom.xml:65]: Done, thanks!

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

**Pull request merged**